dist/js/apps/events-page/app.js

```diff
@@ -967,10 +967,13 @@ var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol
 })(angular.module('eventsPageApp'), bcpl.utility.format);
 'use strict';
 
-(function (app, bootstrapCollapseHelper, onWindowResize, windowShade, globalConstants) {
+(function (app, bootstrapCollapseHelper, onWindowResize, windowShade, globalConstants, googleAnalytics) {
 	'use strict';
 
 	var EventsPageCtrl = function EventsPageCtrl($document, $scope, $timeout, $animate, $location, $window, CONSTANTS, eventsService, filterHelperService, metaService, RequestModel, addthisService) {
+		var trackEvent = googleAnalytics.trackEvent;
+
+
```

Comment:
Extra line skip?

Reply:
@ascire that is a compiled file.

Reply:
Oh.  In that case the transpiler is being sloppy.